### PR TITLE
feat(TGAReader): add vtkTGAReader

### LINKED ISSUE #3155: [Feature] Support for TGA images
### Motivation

Support for the [Truevision Targa](https://fr.wikipedia.org/wiki/Truevision_Targa) (TGA) for textures.

### Detailed Description

Add a new vtkTGAReader

```js
const reader = vtkTGAReader.newInstance();
reader.parse(fileReader.result);
const imageData = reader.getOutputData();
```

### PR DESCRIPTION
**Pull request by daker**
### Context
This PR adds support for TGA format
Fixes #3155

### Results

https://github.com/user-attachments/assets/6b552a7b-6394-4df2-ad04-614c5acf41a8

### Changes
- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

**Comment by finetjul** [comment]
Being curious here, is this still a standard format ? What software still produce/consume file format ?

**Review by finetjul — APPROVED** [review]
LGTM

File: Documentation/content/examples/index.md
Comment (floryst) [inline Documentation/content/examples/index.md:1]: Looks like this file underwent a crlf-lf change. Can you check? We'd want to preserve lf over crlf.

Comment (daker) [reply Documentation/content/examples/index.md:1]: it was crlf, i changed it to lf that's why

**Pull request merged**